frontend/lib/resolution/disambiguation.cpp

```diff
@@ -708,6 +738,198 @@ checkVisibilityInVec(Context* context,
   return MoreVisibleResult::FOUND_NEITHER;
 }
 
+//
+// helper routines for isMoreVisible (below);
+//
+static bool
+isDefinedInBlock(Context* context, const Scope* scope,
+                 const TypedFnSignature* fn) {
+  LookupConfig onlyDecls = LOOKUP_DECLS | LOOKUP_METHODS;
+  //TODO: should this also/only use LOOKUP_INNERMOST?
```

Comment:
no, because then `id == fn->id()` might not match below, because we might not find that match

Reply:
will remove in a later, trivial PR

Reply:
fixed in https://github.com/chapel-lang/chapel/pull/23838